service/src/main/java/uk/gov/hmcts/reform/aac/model/SubmitCallbackResponse.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.aac.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+@ApiModel("SubmitCallbackResponse")
+public class SubmitCallbackResponse {
```

Comment:
is this not the same as the `SubmittedCallbackResponse`?

Reply:
good spot, just copied things from case-assignment repo without veryfying